index.html

```diff
@@ -111,8 +131,8 @@ <h2 class="card-title text-white">Profesional Art Printing Data</h2>
       </div>
       <div class="card bg-image3 child-card">
         <div class="card-body">
-          <h2 class="card-title text-white">Profesional Art Printing Data</h2>
-          <p class="card-description text-white">
+          <h2 class="card-title text-white child-title">Profesional Art Printing Data</h2>
```

Comment:
- [ ] please, kindly make your work responsive once it switches to the desktop version,(the headline, about and contact section.)
![Screenshot (1048)_LI](https://user-images.githubusercontent.com/65298048/236177046-41f7d6bf-fa33-4204-818b-ad0c46e191ef.jpg)